scripts/serve/serve-static.js

```diff
@@ -57,7 +57,8 @@ const virtualApp = (site, index, fs) => {
       filename = `/${index}`;
     }
 
-    const filepath = `${site}${filename}`;
+    // Filter query params
+    const filepath = `${site}${filename}`.replace(/(\?).*/, '');
```

Comment:
This will work on windows. In this case we're removing the '?' query param thats appended to the filename thats coming from the url. I am interested in making sure the virtual filesystem works on windows the same way. Does it switch to windows path separators or stay with linux ones?

Reply:
It does both? I can confirm that running terra-dev-site with this still works on my Windows machine. But the paths I'm getting out of this are 

C:\terra-dev-site\build/index.html
C:\terra-dev-site\build/terra-dev-site-96335e3f6dcb41484c15.css
C:\terra-dev-site\build/raf.js
C:\terra-dev-site\build/babel-polyfill.js
C:\terra-dev-site\build/terra-dev-site.js
C:\terra-dev-site\build/en-translations.js
C:\terra-dev-site\build/204f029ef2a801b1d418d81ffa5be1c1.png